src/auth/authclient.ts

```diff
@@ -668,6 +663,21 @@ export abstract class AuthClient
     }
   }
 
+  /**
+   * Gets the trust boundary header to be attached to the request.
+   */
+  protected getTrustBoundaryHeader(): string | null {
```

Comment:
This could be simplified:

```
if (!this.trustBoundaryEnabled || !this.trustBoundary) {
  return null;
}
// Empty header sent in case trust-boundary has no-op encoded location.
return this.trustBoundary.encodedLocations === NoOpEncodedLocations
  ? ''
  : this.trustBoundary.encodedLocations;